openssl-sys/build.rs

```diff
@@ -217,6 +219,33 @@ fn try_pkg_config() {
     std::process::exit(0);
 }
 
+/// Attempt to find OpenSSL through vcpkg.
+///
+/// Note that if this succeeds then the function does not return as vcpkg
+/// should tell us all the information that we need.
+fn try_vcpkg() {
+
+    // vcpkg will decline if it can not handle the target
+    let lib = match vcpkg::Config::new()
+        .lib_name("libeay32")
```

Comment:
These were renamed to `libcrypto` and `libssl` in OpenSSL 1.1.0 to match up with other platforms. Does vcpkg support OpenSSL 1.1.0 right now?